### PR TITLE
Add `contains_any` parameter to `Text` filter

### PR DESCRIPTION
**Pull request by hoodwinker**
# Description

Text filter extended with "contains_any" parameter. 
"contains" works for all values, "contains_any" for any of.

## New feature

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/dispatcher/filters/builtin.py [hunk 1]
```diff
@@ -268,32 +270,29 @@ def __init__(self,
...
         self.endswith = endswith
         self.startswith = startswith
         self.ignore_case = ignore_case
 
-    @classmethod
-    def validate(cls, full_config: Dict[str, Any]):
-        for param, key in cls._default_params:
-            if param in full_config:
-                return {key: full_config.pop(param)}
-
```
Comment (evgfilim1) [inline aiogram/dispatcher/filters/builtin.py:296]: Why have you deleted this?

Comment (hoodwinker) [reply aiogram/dispatcher/filters/builtin.py:296]: my fault, wrong source

**Review by Olegt0rr — CHANGES_REQUESTED** [review]
You should also add an [example](https://github.com/aiogram/aiogram/blob/dev-2.x/examples/text_filter_example.py)

**Comment by JrooTJunior** [comment]
v2 will no more be supported.

**Pull request closed without merging**